### ISSUE TITLE
Accessibility feedback for Pre-usability review for Left Nav

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7149]
```diff 
! Please note: See feedback from Project kickoff 
! https://github.com/department-of-veterans-affairs/va.gov-team/issues/6751, 
! as well.
```
**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## VSA feedback
- Focus **must** go to the top of the page for each page [focus management].
- As mentioned in the project kickoff feedback, **should** include an indicator of current page for screen readers. This would be accomplished with an `aria-current="page"` on the current page.
- **Consider** a stronger indicator of the on-state, especially for mobile [cognition].
- **Consider** wrapping the left nav in a `<nav>` element with an `aria-label="secondary"` to communicate that this is navigation and distinguish it from the main nav and breadcrumbs.

## VSP feedback
- **Should** use good accordion ARIA markup for the mobile "On this section" button. This includes `aria-expanded="true | false"` and ideally an `aria-controls="UNIQUE_ID"` that points to the navigation container. `aria-controls` only works for JAWS, but it's worth adding as a progressive enhancement.
- **Consider** making the left nav html structure nested `<ol>` + `<li>` with relationships to the previous parent using ARIA:
    ```html
    <li>
      <a href="#" id="link-unique-level-1">Locations</a>
      <ul aria-labelledby="link-unique-level-1">
        <li>
          <a href="#" id="link-unique-level-2">University Drive campus</a>
          <ul aria-labelledby="link-unique-level-2">
            <li>
              <a href="#">Campus map</a>
            </li>
          </ul>
        </li>
      </li>
- **Consider** the design recommendation to change selected element to black text and add the default cursor. We do this on the `aria-current="page"` breadcrumb, turning off pointer-events and defaulting the cursor.

## Definition of done
1. Acknowledge each feedback item with decisions made
2. VSA Accessibility Specialist will close ticket

**Comment by mmiddaugh** [comment]
Feedback for the following have been incorporated into [#7212](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7212)

   - Focus to go to the top of the page
   - Focus state to be consistent with styling across VA.gov. 
   - Indicator of current page
   - Accordion ARIA markup for the mobile "On this section" button.
   - Text for selected objects will be darker text, adding the default cursor
